lib/generators/manageiq/plugin/templates/.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+enableGlobalCache: true
+nodeLinker: node-modules
+yarnPath: .yarn/releases/yarn-3.0.2.cjs
```

Comment:
Instead of hardcoding this value, it might be better to just grab the committed file name, since they must match.  This way, when we go to update it, this will update automatically and we don't have to remember 2 places.